### PR TITLE
Add new scripts to create the services/credentials records locally or remotely

### PR DESCRIPTION
**Pull request by cmoulliard**
- Create a `data` folder to host the scripts able to feed data on Primaza
- Add a new script to create Services locally or remotely
- Moving the script to create a cluster to data

### CODE REVIEW

File: scripts/data/cluster.sh [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 #
 # Usage:
-# ./scripts/create_cluster_entity.sh
+# ./scripts/data/cluster.sh
 #
 # To use a different context
-# CONTEXT_TO_USE=my-ctx ./scripts/create_cluster_entity.sh
+# CONTEXT_TO_USE=my-ctx ./scripts/data/cluster.sh
 #
 # To create the cluster record on a Primaza server which is not localhost:8080
```
Comment (aureamunoz) [inline scripts/data/cluster.sh:10]: I would say: to create the records on a different Primaza server

File: scripts/data/credentials.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+#
+# Usage:
+# ./scripts/data/credentials.sh
+#
+# To create the records on a Primaza server which is not localhost:8080
```
Comment (aureamunoz) [inline scripts/data/credentials.sh:7]: I would say: to create the records on a different Primaza server

File: scripts/data/cluster.sh (hunk 1 above)
Comment (cmoulliard) [reply scripts/data/cluster.sh:10]: Fixed with a70f38c

File: scripts/data/credentials.sh (hunk 2 above)
Comment (cmoulliard) [reply scripts/data/credentials.sh]: Fixed with a70f38c

**Comment by cmoulliard** [comment]
Can I merge it ?

**Pull request merged**